python/test/unit/language/test_core.py

```diff
@@ -2509,8 +2509,8 @@ def test_store_op(M, src_layout, device):
               instr_shape=[16, 8])
 ] if not is_hip() else [
     # TODO (lixun): Add MfmaLayout
-    BlockedLayout([1, 4], [1, 64], [4, 1], [1, 0], [1, 1], [1, 1], [0, 1]),
-    BlockedLayout([1, 4], [1, 64], [2, 2], [1, 0], [1, 1], [1, 1], [0, 1])
+    BlockedLayout([1, 4], [1, THREADS_PER_WARP], [4, 1], [1, 0], [1, 1], [1, 1], [0, 1]),
+    BlockedLayout([1, 4], [1, THREADS_PER_WARP], [2, 2], [1, 0], [1, 1], [1, 1], [0, 1])
```

Comment:
can we merge with the blocked layout above?

Reply:
MmaLayout is also fine for AMD. @zhanglx13 , could we also add it for hip target?

Reply:
> NvidiaMmaLayout is also fine for AMD
@joviliast what do you mean by this, does NvidiaMmaLayout work on AMD ?

Reply:
Is it because this only test layout conversion so it doesn't matter what GPU is used? I'm merging this but please answer @zahimoud's question.

Reply:
Yes, @ThomasRaoux is right.
It could be also pointed out that, common logic from https://github.com/openai/triton/blob/main/lib/Conversion/TritonGPUToLLVM/ConvertLayoutOpToLLVM.cpp#L225
does not requires nvidia specifics from
https://github.com/openai/triton/blob/main/third_party/nvidia/lib/TritonNVIDIAGPUToLLVM/ConvertLayoutOpToLLVM.cpp#L505C3-L505C32 for standalone layout conversions in terms of correctness.

Reply:
I see, but it's kind of counter intuitive, is there a reason to run NvidiaMmaLayout other than just making the testing code cleaner ? If it's just for cleaner testing (i.e. to avoid `if is_hip()` and `if is_cuda()`), we could possibly think of a way to filter layouts without having to add skipping logic in each test ?

Reply:
I agree, we should probably skip those at some point.

Reply:
Ok. It could be done by using different modules for each backend. Something like `compiler.py` .